basics/UserFaq.md

```diff
@@ -219,11 +219,7 @@ or
 
 1.  Go to the sysfs (`/sys/bus/pci`), find the right device, detach it from the pciback driver and attach back to the original driver. Replace `<BDF>` with your device, for example `00:1c.2`:
 
-    ```
     echo 0000:<BDF> > /sys/bus/pci/drivers/pciback/unbind
     MODALIAS=`cat /sys/bus/pci/devices/0000:<BDF>/modalias`
     MOD=`modprobe -R $MODALIAS | head -n 1`
     echo <BDF> > /sys/bus/pci/drivers/$MOD/bind 
```

Comment:
Also note that, if you're going to do space indenting, you need **eight** spaces if the code block is in a list, like this one. Click [here](https://github.com/Jeeppler/qubes-doc/blob/b65aa75876957201b0b43170cdbd6cf7696d273e/basics/UserFaq.md#i-created-a-usbvm-and-assigned-usb-controllers-to-it-now-the-usbvm-wont-boot) to see that the formatting is still not correct. (The other pages look good, though.)